pkg/systemd/page-status.jsx

```diff
@@ -47,20 +49,32 @@ export class PageStatusNotifications extends React.Component {
     }
 
     render() {
-        // Just this one for now
-        const page = "system/services";
-        const status = page_status.get(page);
-        if (status && status.type && status.title) {
-            this.props.toggle_label && this.props.toggle_label(true);
-            const jump = () => cockpit.jump("/" + page);
-            return (
-                <>
-                    <span className={icon_class_for_type(status.type)} />
-                    <a role="button" tabIndex="0" onClick={jump}>{status.title}</a>
-                </>);
-        } else {
-            this.props.toggle_label && this.props.toggle_label(false);
-            return null;
-        }
+        // Explicit whitelist for now, until we can get a dynamic list
+        return ["system/services", "updates"].map(page => {
+            const status = page_status.get(page);
+            if (status && (status.type || status.details) && status.title) {
+                this.props.toggle_label && this.props.toggle_label(true);
+                let target = status.details && status.details.link;
+                if (!target)
+                    target = page;
+                const jump = () => cockpit.jump("/" + target);
+                let action;
+                if (status.type === 'progress')
```

Comment:
Let's not tie "progress" and "not a link" together.  Let's use `link: false` for the latter.

Reply:
Yep, part of the previous comment.